lxd/permissions.go

```diff
@@ -222,8 +224,32 @@ func getPermissions(d *Daemon, r *http.Request) response.Response {
 	}
 
 	if recursion == "1" {
+		slices.SortFunc(apiPermissionInfos, func(a, b api.PermissionInfo) int {
+			if result := cmp.Compare(a.EntityType, b.EntityType); result != 0 {
```

Comment:
I'm reasonably confident that we prefer not to use `if stmt; cmp { ... }` in LXD, could you split the `Compare` to its own line?

Reply:
done 👍